### DISCUSSION TITLE
how to turn off the default generated menu bar

### DISCUSSION THREAD

**Discussion by xiaoqiangclub** [beeware/toga#1476] [Q&A]
hello,
I am a newbie and I would like to ask how to turn off the default generated menu bar (file and Help).
Or how can I change the default 2 menus .
thank you！
![image](https://user-images.githubusercontent.com/61998040/166150441-67267efd-5e59-40c7-9b95-aada7142de6a.png)

**Comment by White-Rabbit-Scientific** [comment]
I too would like to know how to do this. Thanks.

**Comment by freakboy3742** [comment]
This isn't an option that can be turned off, by design. See #733 for discussion about the problem and the complications related to it, and one possible workaround.
